test/src/cpp-integration-rest-schema-evolution.cc

```diff
@@ -0,0 +1,254 @@
+#include <test/support/src/vfs_helpers.h>
```

Comment:
docstring header missing

also is this making `sc-35424.cc` redundant? can we remove it?